### ISSUE TITLE
Bug: Menubar: Third level links don't open on space or enter key

### ISSUE THREAD

**Issue by aldunchev** [NickDJM/accessible-menu#141]
## Summary
When you setup a menu with more than one level deep hierarchy, third level links cannot open with keyboard(enter or space).

## Steps to reproduce
1. Use the menubar plugin
2. Create the following HTML structure:
```
<ul id="example-menu" class="menu">
    <li><a href="/about">About</a></li>
    <li class="dropdown">
      <a href="#">Projects ▼</a>
      <ul class="dropdown-menu">
        <li><a href="/projects/awesome">Awesome project</a></li>
        <li class="dropdown">
          <a href="/projects/not-so-awesome">Not-so-awesome project</a>
          <ul class="dropdown-menu">
            <li><a href="/projects/awesome">Awesome project</a></li>
            <li><a href="/projects/not-so-awesome">Not-so-awesome project</a></li>
          </ul>
        </li>
      </ul>
    </li>
    <li><a href="/contact">Contact me</a></li>
  </ul>
```
3. Try to navigate the menu and hit enter or space with the keyboard. The link does not open.

## Expected Behaviour
Open the link

Here is a video:
https://user-images.githubusercontent.com/4631021/138699357-1218a6f5-e58d-4180-b50b-8e85bf3da948.mov

Here is the test code:
```
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
  <style>
    ul {
      list-style: none;
      padding-left: 0;
    }

    .menu {
      display: flex;
    }

    .menu > li {
      margin: 0 10px;
    }

    .hide {
      display: none;
    }

    .show {
      display: block;
    }

    .dropdown {
      position: relative;
    }

    .dropdown-menu {
      position: absolute;
    }

    .dropdown > .dropdown-menu {
      padding-left: 20px;
    }
  </style>
</head>
<body>
  <ul id="example-menu" class="menu">
    <li><a href="/about">About</a></li>
    <li class="dropdown">
      <a href="#">Projects ▼</a>
      <ul class="dropdown-menu">
        <li><a href="/projects/awesome">Awesome project</a></li>
        <li class="dropdown">
          <a href="/projects/not-so-awesome">Not-so-awesome project</a>
          <ul class="dropdown-menu">
            <li><a href="/projects/awesome">Awesome project</a></li>
            <li><a href="/projects/not-so-awesome">Not-so-awesome project</a></li>
          </ul>
        </li>
      </ul>
    </li>
    <li><a href="/contact">Contact me</a></li>
  </ul>

  <script src="https://cdn.jsdelivr.net/npm/accessible-menu@3.0.0/dist/menubar.min.js"></script>
  <script>
    const menu = new Menubar({
      menuElement: document.querySelector("#example-menu"),
      submenuItemSelector: "li.dropdown",
    });
</script>
</body>

</html>
```

I initially experienced the issue on my development site. Then tested with the dummy html above and managed to reproduce it.
Not sure if it is a bug, misconfiguration or working by design...

Thank you in advance!

**Comment by NickDJM** [comment]
@aldunchev Tested this in my local setup and I can reproduce the error as well. I'll look into what's up and we'll get a fix out.

**Comment by NickDJM** [comment]
This does _not_ happen in disclosure menus or treeviews- looks like it's only an issue for Menubars

**Comment by NickDJM** [comment]
@aldunchev Can you test the change made in #142 and see if that resolves the issue for you? It looks like it did the trick on my end, but would like a second pair of eyes on it.

**Comment by aldunchev** [comment]
Thanks a lot, @NickDJM, that's great! I'll try to test it by end of the day(don't have access to my machine now).

**Comment by aldunchev** [comment]
I confirm that the fix in #142 works. Thanks a lot @NickDJM. I owe you a beer 🍺  🥇

**Comment by NickDJM** [comment]
Excellent! I'll get it merged in and create a release shortly.
